### PR TITLE
Fix/OAP-36/portableelement improvements

### PR DESCRIPTION
**Pull request by oatymart**
This is a non-essential part of https://oat-sa.atlassian.net/browse/OAP-36

- Removed unnecessary lodash usages from EventMgr (replacing with equivalent vanilla JS)
  - This allows PCIs to produce smaller bundles. For example, the `ims/likertScaleInteraction` runtime reduced from 85KB minified to 11KB minified.
  - We could take this further in the hope of achieving more reductions, but this was the easiest win
- Also added a CLI param allowing to run `npx grunt portableelement -e=qtiItemPci -i=likertScaleInteraction -type=IMSPCI`
  - This is useful to target only 1 PCI when you have a duplicated identifier across PCI and IMSPCI types

**To test:** run the unit tests of the PCIs in https://github.com/oat-sa/extension-tao-itemqti-pci, and validate that their behaviour is unchanged

### CODE REVIEW

File: views/build/grunt/portableelement.js [hunk 1]
```diff
@@ -206,7 +206,8 @@ module.exports = function (grunt) {
         const done = this.async();
         const extension = grunt.option('extension') || grunt.option('e');
         const selectedId = grunt.option('identifier') || grunt.option('i');
-        let manifests, compileTasks;
+        const type = grunt.option('type');
```
Comment (jsconan) [inline views/build/grunt/portableelement.js:209]: **suggestion:** add a shortcut as well
```suggestion
        const type = grunt.option('type') || grunt.option('t');
```

File: views/build/grunt/portableelement.js [hunk 2]
```diff
@@ -219,7 +220,9 @@ module.exports = function (grunt) {
             grunt.log.writeln(`Only searching portable element "${selectedId}"`);
         }
 
-        manifests = portableModels.reduce(
+        models = type ? portableModels.filter(model => model.type === type) : portableModels;
```
Comment (jsconan) [inline views/build/grunt/portableelement.js:223]: **suggestion:** Add error handling in case the type mismatches

**Review by jsconan — APPROVED** [review]
Very nice improvement, I like it. Good job!

**Review Checklist**
- [ ] ~~New code is covered by tests (if applicable)~~
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code respects code style rules
- [x] New code respects best practices
- [ ] ~~New code is not subject to concurrency issues (if applicable)~~
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`
- [x] Commits are following [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] Commits messages are meaningful
- [x] Commits are atomic

File: views/build/grunt/portableelement.js (hunk 2 above)
Comment (oatymart) [reply views/build/grunt/portableelement.js:227]: Added in https://github.com/oat-sa/extension-tao-itemqti/pull/2456/commits/d85de92d2c25ddd32d4d3060a2e40ece7eb6e0fe.

**Pull request merged**